crates/hyperswitch_connectors/src/connectors/jpmorgan/transformers.rs

```diff
@@ -142,24 +389,638 @@ impl<F, T> TryFrom<ResponseRouterData<F, JpmorganPaymentsResponse, T, PaymentsRe
+#[serde(rename_all = "camelCase")]
+pub struct PaymentAuthenticationCapReq {}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AuthenticationCapReq {
+    authentication_id: Option<String>,
+    electronic_commerce_indicator: Option<String>,
+    token_authentication_value: Option<String>,
+    s_c_a_exemption_reason: Option<String>,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AccountUpdaterCapReq {
+    request_account_updater: Option<bool>,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ExpiryCapReq {
+    month: i32,
+    year: i32,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct InstallmentCapReq {
+    installment_count: Option<i32>,
+    total_installments: Option<i32>,
+    number_of_deferrals: Option<i32>,
+    plan_id: Option<String>,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RecurringCapReq {
+    recurring_sequence: Option<String>,
+    agreement_id: Option<String>,
+    payment_agreement_expiry_date: Option<String>,
+    is_variable_amount: Option<bool>,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MerchantCapReq {
+    merchant_software: MerchantSoftwareCapReq,
+    merchant_category_code: Option<String>,
+    merchant_logo_url: Option<String>,
+    soft_merchant: SoftMerchantCapReq,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MerchantSoftwareCapReq {
+    company_name: Option<String>,
+    product_name: Option<String>,
+    version: Option<String>,
+    software_id: Option<String>,
+}
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SoftMerchantCapReq {
+    name: Option<String>,
+    phone: Option<String>,
+    email: Option<String>,
+    url: Option<String>,
+    merchant_purchase_description: Option<String>,
+    visa_merchant_verification_value_id: Option<String>,
+    master_card_merchant_verification_value_id: Option<String>,
+    merchant_incorporation_status: Option<String>,
+    foreign_merchant_indicator: Option<bool>,
+}
+
+#[derive(Debug, Default, Copy, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "UPPERCASE")]
+pub enum CapMethod {
+    #[default]
+    Now,
+    Delayed,
+    Manual,
+}
+
+impl TryFrom<&JpmorganRouterData<&PaymentsCaptureRouterData>> for JpmorganCaptureRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: &JpmorganRouterData<&PaymentsCaptureRouterData>,
+    ) -> Result<Self, Self::Error> {
+        let capture_method = item
+            .router_data
+            .request
+            .capture_method
+            .as_ref()
+            .map(|cm| cm.to_string());
```

Comment:
What values are being passed here?

Reply:
in router_data its coming as "manual" and the same will go in the req body of the connector